### PR TITLE
Add internationalization into product summary schema

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?
Add internationalization into product summary schema.

#### What problem is this solving?
This component's shema didn't have the internationalization.

#### How should this be manually tested?
https://waza2--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by lucasecdb** [comment]
could you change the double quotes in the file for single quotes?

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - Added isOneClickBuy attribute.
+- Add internationalization into product summary schema
```
Comment (estacioneto) [inline CHANGELOG.md]: `_ProductSummary_`

File: react/ProductSummary.js [hunk 2]
```diff
@@ -121,51 +121,51 @@ class ProductSummary extends Component {
 }
 
 const defaultSchema = {
-  title: 'Product Summary',
-  description: 'The product summary showing the main product informations',
+  title: 'editor.productSummary.title',
+  description: 'editor.productSummary.description',
   type: 'object',
   properties: {
     showListPrice: {
```
Comment (estacioneto) [inline react/ProductSummary.js]: Usa o schema do `ProductPrice`

no properties:
```javascript
properties: {
  ...Price.properties,
  // ...
}
```

File: react/locales/en-US.json [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+  "editor.productSummary.description": "The product summary showing the main product informations",
+  "editor.productSummary.showListPrice.title": "Show product's list price",
+  "editor.productSummary.isOneClickBuy.title": "Should redirect to checkout after clicking on buy",
+  "editor.productSummary.showLabels.title": "Show product's prices' labels",
+  "editor.productSummary.showInstallments.title": "Show product's payment installments",
+  "editor.productSummary.showBadge.title": "Show the discount badge",
+  "editor.productSummary.badgeText.title": "Badge's text",
+  "editor.productSummary.buyButtonText.title": "Custom buy button's text",
+  "editor.productSummary.hideBuyButton.title": "Hides the buy button completely",
+  "editor.productSummary.showButtonOnHover.title": "Show the buy button only on hover"
```
Comment (estacioneto) [inline react/locales/en-US.json]: Atualiza tirando as propriedades do Price

**Pull request merged**